README.md

```diff
@@ -3,38 +3,58 @@
 A client library to access the not-so-public Netflix Shakti API.
 
 ## Installation
-```
+```bash
 npm install netflix2
 ```
 
 ## Usage
 All functions take standard Node callbacks:
-```
+```javascript
 function callback (error, result) {}
 ```
 
 ### Initialization
-```
+```javascript
 var Netflix = require('netflix2')
 var netflix = new Netflix()
 ```
 or
-```
+```javascript
 var netflix = require('netflix2')()
 ```
 
 ### Login
 You must call `login` before using any of the other below functions. This will set cookies, API endpoints, and the authURL that must used to make API calls.
-```
+```javascript
 var credentials = {
   email: 'youremail@example.com'
   password: 'yourpassword'
 }
 netflix.login(credentials, callback)
 ```
 
+### Browse
+```javascript
+/**
+ * Browse movies, to simply get all films use Category ID 34399
+ *
+ * @param genreId The Netflix Category ID, Like https://www.netflix.com/browse/genre/34399
+ * @param page The content is paged, this is the page number.
+ * @param perPage How many items do you want per page?
+ * @param callback Function to be called when the request is finished.
+ */
+netflix.browse(genreId, page, perPage, function (error, result) {
+  if(error){
+    console.error(error);
+  }else{
```

Comment:
Just a style thing, but I like spaces around my `else`s. If you'd like, you could add that rule to the eslint config file, but you don't have to.

Reply:
No problem. Let me know if you prefer the same spacing for the if argument.
As long as there's a clear way for consistent text formatting I'm happy to follow.